google-cloud-aiplatform/src/main/java/com/google/cloud/aiplatform/v1beta1/utility/ValueConverter.java

```diff
@@ -0,0 +1,48 @@
+package com.google.cloud.aiplatform.v1beta1.utility;
```

Comment:
Needs a license header

Reply:
Also, should this be in an unversioned namespace (will v1beta1's be different than v1's)?

Reply:
Done.